packages/orbit-components/src/Accordion/AccordionSection/components/SectionHeader.tsx

```diff
@@ -73,12 +64,14 @@ const AccordionSectionHeader = ({
         isInteractive && "hover:bg-cloud-light cursor-pointer border-0 bg-transparent text-left",
       )}
       data-test={dataTest && `${dataTest}Header`}
-      aria-expanded={expanded}
+      aria-labelledby={ariaLabelledby}
       {...(isInteractive && {
         role: "button",
         onClick: handleClick,
         onKeyDown: handleKeyDown,
         tabIndex: 0,
+        "aria-controls": ariaControls,
+        "aria-expanded": expanded || undefined,
```

Comment:
Is this working correctly? On [this story](https://kiwicom-orbit-sarka-accordion-a11y-fix.surge.sh/?path=/story/accordion--mobile-first-interaction), I don't see aria-controls being applied

Reply:
what do you mean? value of `aria-controls` is meant to be used internally (I removed the arg from SB already). I can see this attribute with the value in DOM before it's opened.